sky/utils/subprocess_utils.py

```diff
@@ -113,6 +114,12 @@ def run_in_parallel(func: Callable,
       A list of the return values of the function func, in the same order as the
       arguments.
     """
+    if isinstance(args, collections.abc.Sized):
```

Comment:
nit: how about we change the input args to be `List[Any]` and change the invocations to use lists to avoid this `collections.abc.Sized`?

Reply:
Sounds good, I will try this~